.github/workflows/test.yml

```diff
@@ -366,6 +366,11 @@ jobs:
             ~/.cache/go-build
           key: go-build-${{ hashFiles('**/go.sum') }}
           restore-keys: go-build-
+      - name: Set current branch
+        run: |
+          raw=$(git branch -r --contains ${{ github.ref }})
+          branch=${raw##*/}
+          echo "BRANCH=$branch" >> $GITHUB_ENV
```

Comment:
I suggest naming the variable `CURRENT_BRANCH`, so that it aligns with the existing constants. For example:

```
const (
	branch        = "gh-pages"
	tokenKey      = "GITHUB_TOKEN"
	currentBranchKey = "CURRENT_BRANCH"
	defaultBranch = "main"
)
```

Reply:
Yep, makes sense. Renamed as suggested :)